src/Concordium/Client/Config.hs

```diff
@@ -841,11 +841,7 @@ getAccountConfig :: Maybe Text
                  -- This will also ask the user for an optional name mapping.
                  -> IO (BaseConfig, AccountConfig)
 getAccountConfig account baseCfg keysDir keyMap encKey autoInit = do
-  account' <- case account of
-    Nothing -> do
-      logInfo [printf "account reference not provided; using \"%s\"" defaultAccountName]
-      return defaultAccountName
-    Just a -> return a
+  account' <- getAccountOrDefault account
   namedAddr <- case getAccountAddress (bcAccountNameMap baseCfg) account' of
                  Left err -> logFatal [err]
```

Comment:
Perhaps it would be useful to revise the error message here in case we attempted to use the "default" account.
We could make it less confusing.